schedule/yam/agama_software_selection_unattended.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: agama_software_selection_unattended
+description: |
+  - Prepare url for agama.auto boot parameterer form auto-installation.
```

Comment:
drop this, makes no sense, it is not specific from this test suite, you could copy/paste the enhanced description from job group, same for below file.

Reply:
Dropped this line for both files. Thanks.